ui/shared/ad/CustomAdBanner.tsx

```diff
@@ -0,0 +1,48 @@
+import { Flex, chakra, Tooltip, Image } from '@chakra-ui/react';
+import React, { useState, useEffect } from 'react';
+
+import useCustomBanners from 'ui/shared/ad/useCustomBanners';
+
+const CustomAdBanner = ({ className }: { className?: string }) => {
+  const customBanners = useCustomBanners().data || [];
+  const [ currentBannerIndex, setCurrentBannerIndex ] = useState(0);
+  useEffect(() => {
+    if (customBanners.length === 0) {
+      return;
+    }
+    const timer = setInterval(() => {
+      setCurrentBannerIndex((prevIndex) => (prevIndex + 1) % customBanners.length);
+    }, 60000);
+
+    return () => {
+      clearInterval(timer);
+    };
+  }, [ customBanners.length ]);
+
+  if (customBanners.length === 0) {
+    return null;
+  }
+
+  const currentBanner = customBanners[currentBannerIndex];
+
+  return (
+    <>
+      <Flex className={ className } h="90px" display={{ base: 'none', lg: 'flex' }}>
+        <Tooltip label={ currentBanner.text } aria-label={ currentBanner.text }>
+          <a href={ currentBanner.url } target="_blank" rel="noopener noreferrer">
+            <Image src={ currentBanner.desktop } alt={ currentBanner.text } height="100%" width="auto" borderRadius="10px"/>
```

Comment:
let's use one of our standard values for `border-radius` here - `theme/foundations/borders.ts`

Reply:
Can you tell me how to use this? How to replace borderRadius="10px"?

Reply:
Well, I've found it.
  return (
    <Flex className={ className } h="90px">
      <Tooltip label={ currentBanner.text } aria-label={ currentBanner.text }>
        <a href={ currentBanner.url } target="_blank" rel="noopener noreferrer">
          <Image src={ isMobile ? currentBanner.mobile : currentBanner.desktop } alt={ currentBanner.text } height="100%" width="auto" borderRadius="md"/>
        </a>
      </Tooltip>
    </Flex>
  );